pulp_2_tests/tests/rpm/api_v2/test_modularity.py

```diff
@@ -49,6 +50,13 @@ def setUpModule():  # pylint:disable=invalid-name
         raise unittest.SkipTest('https://pulp.plan.io/issues/4405')
 
 
+def tearDownModule():  # pylint:disable=invalid-name
+    """Delete the orphans created by testing."""
+    cli.Client(config.get_config()).run((
+        'pulp-admin', 'orphan', 'remove', '--all'
+    ))
```

Comment:
Why do we need to call clean orphans? Any other change?

Reply:
@kersommoura before I started making any other changes, I ran the test and it was leaving a lot of orphans around. Therefore, I decided to add a module wide tear down so `test_modularity` didn't leave orphans on the server.

I can re-test to ensure that is the case, if you like.

Reply:
@bherrin3, if there is no strong need to call clean orphans, we should not call it.